api/app/services/ingestor/strategy/google_doc/inspector/google_doc.rb

```diff
@@ -63,6 +63,13 @@ def ingestion_source
             [ingestion_source_path]
           end
 
+          def spine_ids
+            ingestion_source.map do |item|
+              ::Ingestor::Strategy::Gitbook::Inspector::TextSection.new(item, self)
+                                                                   .source_identifier
+            end
+          end
+
           # rubocop:disable Metrics/LineLength
           def ingestion_source_inspectors
             ingestion_source.map do |source|
```

Comment:
Also not part of the PR.. just seems better to do `[ingestion_source].map` here and have ingestion_source return one thing.